source/2020-01-27-the-ember-times-issue-132.md

```diff
@@ -46,13 +46,13 @@ We encourage you to watch the video and share your own applications of `metadata
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Related Files Hopper VSCode Extension 🤸](https://twitter.com/suchita009/status/1218310260592496641)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Ever needed to keep finding related files in your Ember repo using [VSCode](https://code.visualstudio.com/)? [Suchita Doshi (@suchitadoshi1987)](https://github.com/suchitadoshi1987) created an extension to make that leap easier!
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The [Related Files Hopper extension](https://marketplace.visualstudio.com/items?itemName=suchitadoshi1987.file-hopper) helps developers navigate through the multiple files that are touched when working on a feature. Within Ember for example, a person adding or removing something in `components/foo.js` would likely also touch `templates/foo.hbs` and related tests `unit/foo-test.js` or `acceptance/foo-test.js`. This extension provides hot keys to present related files as an option to jump into while you’re currently working in a selected file.
```

Comment:
Hmm, maybe suggest using a rendering test for testing a component, so that a reader doesn't need to think that the developer, for example, may have extracted methods into a utility for unit testing?

 In the last sentence, I used your component file to make your idea more concrete. I think my edits convey what you had meant, but just in case, can you check this? Thanks.

```suggestion
The [Related Files Hopper extension](https://marketplace.visualstudio.com/items?itemName=suchitadoshi1987.file-hopper) helps developers navigate through the multiple files that are touched when working on a feature. Within Ember for example, a person adding or removing something in `components/foo.js` would likely also touch `templates/foo.hbs` and related tests `integration/foo-test.js` or `acceptance/foo-test.js`. This extension provides hot keys for related files when you're working in `components/foo.js`.
```

Reply:
ah yeah good point copied the example on github :p